src/libtls/tls_server.c

```diff
@@ -701,12 +975,73 @@ static status_t send_server_hello(private_tls_server_t *this,
 	/* NULL compression only */
 	writer->write_uint8(writer, 0);
 
+	if (version >= TLS_1_3)
+	{
+		bio_writer_t *ext_content;
+		extensions = bio_writer_create(32);
```

Comment:
I guess `extensions` could also be declared here as it it not used in anywhere else in this function, or declare both at the start of the function. And again, please add an empty line after the declarations in larger blocks.